tests/__snapshots__/test_help_messages.ambr

```diff
@@ -3008,59 +3008,66 @@
+  │ *  --spec-path                             FILE             Path to service  │
+  │                                                             specification    │
+  │                                                             file.            │
+  │                                                             [default: None]  │
+  │                                                             [required]       │
+  │    --min-instances                         INTEGER RANGE    Minimum number   │
+  │                                            [x>=1]           of service       │
+  │                                                             instances to     │
+  │                                                             run.             │
+  │                                                             [default: 1]     │
+  │    --max-instances                         INTEGER RANGE    Maximum number   │
+  │                                            [x>=1]           of service       │
+  │                                                             instances to     │
+  │                                                             run.             │
+  │                                                             [default: None]  │
+  │    --auto-resume        --no-auto-resu…                     The service will │
+  │                                                             automatically    │
+  │                                                             resume when a    │
+  │                                                             service function │
+  │                                                             or ingress is    │
+  │                                                             called.          │
+  │                                                             [default:        │
+  │                                                             auto-resume]     │
+  │    --eai-name                              TEXT             Identifies       │
+  │                                                             External Access  │
+  │                                                             Integrations(EA… │
+  │                                                             that the service │
+  │                                                             can access. This │
+  │                                                             option may be    │
+  │                                                             specified        │
+  │                                                             multiple times   │
+  │                                                             for multiple     │
+  │                                                             EAIs.            │
+  │                                                             [default: None]  │
+  │    --query-wareho…                         TEXT             Warehouse to use │
```

Comment:
Is the `--query-wareho…`  expected?

Reply:
yep in console works fine, I believe the test runner will just truncate names that are too long based on its window
<img width="1566" alt="image" src="https://github.com/Snowflake-Labs/snowflake-cli/assets/156019931/bed2fbdd-6091-4c48-9af9-3adb8342337f">